pkg/cloud/azure/actuators/machine/actuator.go

```diff
@@ -123,6 +126,7 @@ func (a *Actuator) Update(ctx context.Context, cluster *clusterv1.Cluster, machi
 
 // Exists test for the existence of a machine and is invoked by the Machine Controller
 func (a *Actuator) Exists(ctx context.Context, cluster *clusterv1.Cluster, machine *clusterv1.Machine) (bool, error) {
+	defer actuators.Recover()
```

Comment:
Helps debugging panics in controller, added across all controllers Create/Update/Exists/Delete